test/EndToEnd/GivenDotNetUsesMSBuild.cs

```diff
@@ -9,6 +9,10 @@ namespace Microsoft.DotNet.Tests.EndToEnd
 {
     public class GivenDotNetUsesMSBuild : TestBase
     {
+        public static void Main() 
```

Comment:
Isn't it just as small to remove `"emitEntryPoint": true` from the project.json and then we don't have to worry about this anymore?

---
In reply to: [83269444](https://github.com/dotnet/cli/pull/4382#discussion_r83269444) [](ancestors = 83269444)